Java/syntax_test_java.java

```diff
@@ -122,9 +126,10 @@ private static void anotherMethod() throws MyException<Abc> {
     }
 
     <T> void save(T obj);
-//  ^^^^^^^^^^^^^^^^^^^^ meta.method
+//           ^^^^^^^^^^^ meta.method
 //  ^^^ meta.generic
-//      ^^^^ meta.method.return-type
+//   ^  variable.parameter.type
```

Comment:
This would be a new addition to our scope guidelines. I don't know if for C++, Rust, Java it is clear in what circumstances this scope would be applied. I am certainly no expert in the finer points of templating, so some one else may have more insight or feel more strongly about how this should be handled.

EDIT: Just saw a comment below from myself from back on Jun 8 about an example father down. 😄